tagging.py

```
@@ -1,16 +1,503 @@
+            lex.backup()
+            break
+    word = lex.input[lex.start : lex.pos]
+
+    lower_word = word.casefold()
+    if lower_word in key:
+        token_type = key[lower_word]
+        if token_type in (ItemType.Honorific, ItemType.InfoSpecifier):
+            lex.accept(".")
+        lex.emit(token_type)
+    elif cal(word):
+            lex.emit(ItemType.Calendar)
```

Flake8 (7.2.0)

[error] over-indented

(E117)